src/http_rest_api_handler.cpp

```diff
@@ -443,29 +437,29 @@ Status HttpRestApiHandler::dispatchToProcessor(
     return StatusCode::UNKNOWN_REQUEST_COMPONENTS_TYPE;
 }
 
-Status HttpRestApiHandler::processV3(const std::string_view uri, const HttpRequestComponents& request_components, std::string& response, const std::string& request_body, tensorflow::serving::net_http::ServerRequestInterface* serverReaderWriter) {
+Status HttpRestApiHandler::processV3(const std::string_view uri, const HttpRequestComponents& request_components, std::string& response, const std::string& request_body, std::shared_ptr<HttpAsyncWriter> serverReaderWriter) {
 #if (MEDIAPIPE_DISABLE == 0)
     OVMS_PROFILE_FUNCTION();
     HttpPayload request;
-    Document doc;
+    std::shared_ptr<Document> doc = std::make_shared<Document>();
```

Comment:
Because when using drogon, we do not processV3 exits early, before the request is handled. The workload is moved to thread pool, and when using reference, it would be destroyed earlier than required.